pkg/ccl/crosscluster/physical/standby_read_ts_poller_job_test.go

```diff
@@ -89,3 +91,152 @@ INSERT INTO a VALUES (1);
 		return nil
 	}, time.Minute)
 }
+
+func TestReaderTenantWhenCutoverToLatest(t *testing.T) {
```

Comment:
could these 2 tests get combined into one? they share basically the same code, except post cutover:
- if latest: check the reader can still serve reads
- if not latest: check the reader is shut down.

An elegant way to do this would be to use the `t.RunTrueFalse` helper, e.g.:
https://github.com/azhu-crl/cockroach/blob/reader-tenant-fast-failback/pkg/ccl/crosscluster/logical/logical_replication_job_test.go#L1269